lib/zendesk_apps_support/package.rb

```diff
@@ -121,6 +121,31 @@ def translation_files
       files.select { |f| f =~ %r{^translations/} }
     end
 
+    def obj(options)
```

Comment:
So this is called on build here: https://github.com/zendesk/zendesk_app_market/pull/3911/files#diff-f79896725665a803c7ae9d0128e75729R45.  Maybe this name should be something a bit clearer, such as `build_object`?

Reply:
Or `build_object_metadata`?

Reply:
... or `set_app_metadata`